jpamodelgen dependency must be updated

shadai-rafael opened the pull request:
Dependency have to be updated to

<annotationProcessorPaths>
    <path>
        <groupId>org.hibernate.orm</groupId>
        <artifactId>hibernate-jpamodelgen</artifactId>
    </path>
</annotationProcessorPaths>

sdelamo committed:
ci: projectVersion=4.8.0-SNAPSHOT [ci skip]

sdelamo committed:
Merge branch '4.7.x' into 4.8.x

renovate[bot] committed:
fix(deps): update spring data (#2877)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency io.micronaut.spring:micronaut-spring-bom to v5.6.0 (#2878)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update gradle/gradle-build-action action to v3.3.0 (#2869)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

dstepanov committed:
Rollback PRs intended for 4.8.x (#2882)

dstepanov committed:
Support fetching non-unique ID entities without JOINs (#2883)

dstepanov committed:
Revert "Rollback PRs intended for 4.8.x (#2882)" (#2890)

This reverts commit ecc177ca97d6a77130e6f7a7cef131bdc17d203c.

renovate[bot] committed:
fix(deps): update dependency org.springframework.boot:spring-boot-gradle-plugin to v3.2.5 (#2897)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update gradle/gradle-build-action action to v3.3.1 (#2895)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update gradle/wrapper-validation-action action to v3 (#2880)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency org.apache.groovy:groovy-dateutil to v4.0.21 (#2874)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency io.micronaut:micronaut-core-bom to v4.4.6 (#2894)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

dstepanov committed:
Reproduce query escape (#2900)

micronaut-build committed:
Update common files (#2881)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/micronaut-projects/micronaut-data?pullRequest=2907) <br/>All committers have signed the CLA.

radovanradic commented on src/main/docs/guide/dbc/dbcCriteriaSpecifications/typeSafeJava.adoc:
```diff
@@ -19,7 +19,7 @@ Note that as of this writing you cannot use Micronaut Data annotations (those fo
-dependency:hibernate-jpamodelgen-jakarta[groupId="org.hibernate", scope="annotationProcessor"]
+dependency:hibernate-jpamodelgen[groupId="org.hibernate.orm", scope="annotationProcessor"]
 
 NOTE: The Hibernate 6 version of `hibernate-jpamodelgen-jakarta` is required because prior versions of Hibernate are still using the `javax.persistence` package
```
Please changes also 
```The Hibernate 6 version of `hibernate-jpamodelgen-jakarta` ```
to
```The Hibernate 6 version of `hibernate-jpamodelgen` ```

dstepanov replied:
NOTE should be removed at this point

shadai-rafael replied:
@dstepanov @radovanradic I am having problems to commit a new patch following your comments.
I am not able to push another commit, I am rejected. So I synchronize my fork as it says in the next link: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-command-line but instead of main I am merging with 4.7.x, I got the next result:

68048a45c (HEAD -> patch-1) Update typeSafeJava.adoc
2c5dd3b447 (upstream/4.7.x) docs: clarify how to add annotation processor dependencies for Kotlin and Groovy projects. (#2910)
2628b21e1b (origin/4.7.x) Rollback PRs intended for 4.8.x (#2889)
5c49c0dd8e Support system/pseudo column as version field (ORA_ROWSCN) (#2873)

but still I am rejected, could you please let me know what I am doing wrong?

radovanradic replied:
Don't know what could be wrong, maybe would be easier to close this PR and open a new one against 4.8.x.

radovanradic committed:
Minor Improvement for Oracle JsonView optimistic locking (#2915)

* Minor Improvement for Oracle JsonView optimistic locking

Extract SqlExceptionMapper for extensibility.
Added support for optimistic lock checking in batch update for JsonView.

* Separate handlers for Jdbc and R2dbc

* Address review comments.

* Removed unused var and imports

* More changes per CR comments

* Sonar suggested changes

sdelamo committed:
replace .collect(Collectors.toList()) with toList (#2905)

* replace .collect(Collectors.toList()) with toList

> Stream.toList() provides a List implementation that is immutable (type ImmutableCollections.ListN that cannot be added to or sorted) similar to that provided by List.of() and in contrast to the mutable (can be changed and sorted) ArrayList provided by Stream.collect(Collectors.toList()).

* Fix unused and star imports

* Fix type errors when moving to toList()

By reverting back to collect

* More unused imports

---------

Co-authored-by: Tim Yates <tim.yates@gmail.com>

radovanradic committed:
Add test and reflection metadata for Hibernate Reactive ManyToMany and ElementCollection (#2919)
